src/appengine/libs/handler.py

```diff
@@ -161,6 +161,27 @@ def wrapper(self, *args, **kwargs):
   return wrapper
 
 
+def _validade_access_token(authorization):
+  access_token = authorization.split(' ')[1]
+  response_access_token = requests.get(
+      'https://www.googleapis.com/oauth2/v3/tokeninfo',
+      params={'access_token': access_token},
+      timeout=HTTP_GET_TIMEOUT_SECS)
+  response_id_token = requests.get(
```

Comment:
can we short circuit this if the access token response succeeds? i.e. we don't make the id token request.